src/prop-types/ref.js

```diff
@@ -1,5 +1,3 @@
 import PropTypes from 'prop-types';
 
-export default PropTypes.shape({
-  value: PropTypes.any,
-});
+export default PropTypes.object;
```

Comment:
Figured that `PropTypes.object` makes sense here because:

1. Refs have a `current` property, not `value`, [per the docs](https://reactjs.org/docs/refs-and-the-dom.html#accessing-refs).
2. This never failed, though, because the value key here is not required.
3. We can't make it required, because sometimes the `current` property will be null.

Because of all these things, it doesn't seem that we can enforce much about the `current` property without a custom prop type, which probably isn't worth it.

Reply:
Nice! I was thinking of just switching our `value` to `current`. But this makes more sense. I figured we have a lint rule that errors on ambiguous property types -- but maybe not?

Reply:
Changing value to current would totally make sense. In this case, I think that the combination

- `shape`
- Non-required keys
- PropTypes.any

Results in it basically being the same as `PropTypes.object`.

Using `shape` would have documentation benefits, though, and could be worth switching back to.